src/components/07-form/templates/address-form.njk

```diff
@@ -1,11 +1,19 @@
 <form class="usa-form usa-form--large">
   <fieldset class="usa-fieldset">
     <legend class="usa-legend">Mailing address</legend>
-    <label class="usa-label" for="mailing-address-1">Street address 1</label>
+    <label class="usa-label" for="mailing-address-1">Street Number and Name</label>
     <input class="usa-input" id="mailing-address-1" name="mailing-address-1" type="text">
 
-    <label class="usa-label" for="mailing-address-2">Street address 2 <span class="usa-hint">(optional)</span></label>
-    <input class="usa-input" id="mailing-address-2" name="mailing-address-2" type="text">
+    <label class="usa-label" for="apt-floor-suite">Apartment/Floor/Suite <span class="usa-hint">(optional)</span></label>
+    <select class="usa-select" id="apt-floor-suite" name="apt-floor-suite">
+        <option value>- Select One -</option>
```

Comment:
```suggestion
        <option value>- Select -</option>
```
Should this match the `select` [in our guidance](https://designsystem.digital.gov/components/form-controls/#dropdown)?